source/server/watchdog_impl.h

```diff
@@ -24,21 +18,26 @@ class WatchDogImpl : public WatchDog {
    */
   WatchDogImpl(int32_t thread_id, SystemTimeSource& tsource, std::chrono::milliseconds interval)
       : thread_id_(thread_id), time_source_(tsource),
-        latest_touch_time_(tsource.currentSystemTime()), timer_interval_(interval) {}
+        latest_touch_time_since_epoch_(tsource.currentSystemTime().time_since_epoch()),
+        timer_interval_(interval) {}
 
-  int32_t threadId() const { return thread_id_; }
-  SystemTime lastTouchTime() const { return latest_touch_time_.load(); }
+  int32_t threadId() const override { return thread_id_; }
+  SystemTime lastTouchTime() const override {
+    return SystemTime(latest_touch_time_since_epoch_.load());
+  }
 
   // Server::WatchDog
   void startWatchdog(Event::Dispatcher& dispatcher) override;
-  void touch() override { latest_touch_time_.store(time_source_.currentSystemTime()); }
+  void touch() override {
+    latest_touch_time_since_epoch_.store(time_source_.currentSystemTime().time_since_epoch());
+  }
 
 private:
   const int32_t thread_id_;
   SystemTimeSource& time_source_;
-  std::atomic<SystemTime> latest_touch_time_;
+  std::atomic<std::chrono::system_clock::duration> latest_touch_time_since_epoch_;
   Event::TimerPtr timer_;
   const std::chrono::milliseconds timer_interval_;
 };
 
-} // Server
+} // namespace Server
```

Comment:
This is actually my clang-format doing it automatically. According to https://clang.llvm.org/docs/ClangFormatStyleOptions.html we can flip this on with FixNamespaceComments.